scripts/undocumented_classes.txt

```diff
@@ -50,4 +50,6 @@ com.mapbox.navigation.ui/OnNavigationReadyCallback
 com.mapbox.navigation.ui/RecenterBtnClickListener
 com.mapbox.navigation.ui/RouteOverviewBtnClickListener
 com.mapbox.navigation.ui/SoundButton
-com.mapbox.navigation.ui/SummaryBottomSheetCallback
\ No newline at end of file
+com.mapbox.navigation.ui.speed/SpeedLimitView
+com.mapbox.navigation.ui/SummaryBottomSheetCallback/onStateChanged/#android.view.View#int
+com.mapbox.navigation.ui/SummaryBottomSheetCallback/onSlide/#android.view.View#float
```

Comment:
replacing `com.mapbox.navigation.ui/SummaryBottomSheetCallback` by 
```
com.mapbox.navigation.ui/SummaryBottomSheetCallback/onStateChanged/#android.view.View#int
com.mapbox.navigation.ui/SummaryBottomSheetCallback/onSlide/#android.view.View#float
```
is not needed

Reply:
fails for me locally without it, do you know what can cause this?

Reply:
argh, sorry, I replaced that line instead of creating a new one